### ISSUE TITLE
Android - b2cClient.isSignedIn() always return false

### ISSUE THREAD

**Issue by Maddumage** [stashenergy/react-native-msal#65]
I have updated the library to the latest version ("react-native-msal": "^3.0.5"). After updating to the latest isSignedIn() method always returns false. I get the latest updated code from the b2c sample. Previously this worked well. Is there any issue on my side, or else?.

**Comment by josmithua** [comment]
It should be working. Maybe check if you have the latest copy of `b2cClient.ts`, I know I made some changes since I first published it. If everything checks out there, can you try reproducing this in the example app of this repo?

Cheers

**Comment by Maddumage** [comment]
Yes, I have updated the b2cClient.ts file from the sample. Let me test it again and try the example app of this repo.
Thanks

**Comment by Maddumage** [comment]
Hi @josmithua, I found the issue that was on my side and not in the library. The reason to get this issue was that mismatching the sign-in sign-up policy name which I have used to log in. I recently changed my custom policy but I have forgotten to update the msal_config.json file with the latest policy name.
After making the changes for msal_config this worked well.
Thank you very much for your guidance.
